src/main/java/org/fundacionjala/at13/spaceinvaders/Bullet.java

```diff
@@ -63,8 +55,7 @@ public void changeBulletExists(final Boolean newState) {
     }
 
     /**
-     * Description: The function displays a bullet shoot.
-     * from the Spaceship.
+     * Description: The function displays a bullet shoot. from the Spaceship.
```

Comment:
it would be better if you remove the point between short and example Description: The function displays a bullet shoot from the Spaceship.

Reply:
okay, already solved 😁